types/screeps-arena/package.json

```diff
@@ -18,10 +18,6 @@
         {
             "name": "Skyler Kehren",
             "githubUsername": "pyrodogg"
-        },
-        {
-            "name": "Autumn Saury",
-            "githubUsername": "AutumnSaury"
```

Comment:
@Astrosake is this your old github username? Would you like to stay an owner of screeps-arena?